src/Symfony/Component/Cache/Adapter/RedisTagAwareAdapter.php

```diff
@@ -279,6 +281,11 @@ private function getRedisEvictionPolicy(): string
         }
 
         foreach ($this->getHosts() as $host) {
+            if ($host instanceof ClientInterface && $host->getConnection() instanceof ReplicationInterface && method_exists($host, 'getClientFor')) {
+                // As discussed (https://github.com/predis/predis/issues/598#issuecomment-671876712) predis won't allow any
```

Comment:
please remove the link to github (same below), we don't track this in the source (git[hub] history FTW)